CHANGELOG.md

```diff
@@ -9,19 +9,10 @@
 
 # UNRELEASED
 
-# Node and Miner v1.32.0-rc3 / 2025-03-19
+# Node and Miner v1.32.0 / 2025-03-27
 
-This is the third release candidate of the **upcoming MANDATORY Lotus v1.32.0 release**, which will deliver the Filecoin network version 25, codenamed “Teep” 🦵. This release candidate sets the upgrade epoch for the calibration network to **Epoch 2523454 - 2025-03-26T23:00:00Z**. This is equivalent to:
+This is the stable release of the **upcoming MANDATORY Lotus v1.32.0 release**, which will deliver the Filecoin network version 25, codenamed “Teep” 🦵. This release candidate sets the upgrade epoch for the Mainnet to **Epoch 4867320 - 2025-04-10T23:00:00Z**. This is equivalent to:
```

Comment:
Remove "This is equivalent to:"

Reply:
I removed this from the release here: https://github.com/filecoin-project/lotus/releases/tag/v1.32.0